packages/babel-parser/test/fixtures/es2020/optional-chaining/new-ts-createParenthesizedExpressions-false/input.ts

```diff
@@ -0,0 +1,7 @@
+new foo?.bar();
+new foo.bar?.();
+new foo?.bar!()
+
+new (foo?.bar)();
+foo?.[0]
+foo![0]
```

Comment:
Shouldn't we test `new foo?.[0];` and `new foo![0];` instead?

Reply:
oops. :)